spec/features/eventmanager_spec.rb

```diff
@@ -1,63 +1,67 @@
 require 'rails_helper'
 require 'database_cleaner'
 
-RSpec.feature "CreateEvents", type: :feature do
-Capybara.default_driver = :selenium
-# Capybara.current_driver = :webkit
-
-
-    before do
-      # page.driver.browser.manage.window.resize_to(990,640)
+RSpec.feature "Event Manager abilities", type: :feature, js: true do
+    before :all do
       OmniAuth.config.test_mode = true
       Category.create({name: 'Alex', description: "Alex's special category for crappy things"})
       Category.create({name: 'Music', description: "music's special category for crappy things"})
     end
-
     after do
       DatabaseCleaner.clean
     end
 
-    scenario "User can create events" do
+    scenario "user wants to become an Event Manager" do
       visit root_path
 
       click_link 'Log In'
 
-      click_link 'Google'
+      within ".modal-content" do
+        click_link 'Google'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.